solidity/test/bridge/Bridge.Parameters.test.ts

```diff
@@ -444,11 +451,40 @@ describe("Bridge - Parameters", () => {
                 constants.movedFundsSweepTimeoutNotifierRewardMultiplier
               )
           ).to.be.revertedWith(
-            "Moving funds dust threshold must be greater than zero"
+            "Moving funds dust threshold must be greater than zero and smaller than redemption dust threshold"
           )
         })
       })
 
+      context(
+        "when new moving funds dust threshold is not smaller than redemption dust threshold",
```

Comment:
```suggestion
        "when new moving funds dust threshold is not lower than redemption dust threshold",
```

Reply:
Done in https://github.com/keep-network/tbtc-v2/pull/241/commits/f2a5d262f26294ddcbdaf4295aa33b3cc18a5418.